source/documentation/standards/naming-domains.html.md.erb

```diff
@@ -43,6 +42,19 @@ Office365), or on which MOJ has a corporate presence
 [Trello](https://trello.com/mojds/home)) are not required to be served
 from a `*.service.justice.gov.uk` subdomain
 
+#### Cloud and Modernisation Platform subdomains
+
+If you're using the Cloud or Modernisation Platform to host your service, you will get a subdomain
```

Comment:
Maybe add something about these not being for production use, and to work with the relevant team and the domains team to get your final domain?